src/app/components/profile-membership/profile-membership.component.ts

```diff
@@ -1,26 +1,30 @@
 import { Component, OnInit } from '@angular/core';
 import { UserService } from 'src/app/services/user-service/user.service';
 import { User } from 'src/app/models/user';
+import { Car } from 'src/app/models/car';
+import { CarService } from 'src/app/services/car-service/car.service';
 @Component({
   selector: 'app-profile-membership',
   templateUrl: './profile-membership.component.html',
   styleUrls: ['./profile-membership.component.css']
 })
 export class ProfileMembershipComponent implements OnInit {
-  profileObject : User;
-  currentUser: any = '';
-  isDriver: boolean;
-  active: boolean;
+  profileObject: User;
+  carObject: Car;
   success: string;
-  constructor(private userService: UserService) { }
+
+  constructor(private userService: UserService, private carService: CarService) { }
   ngOnInit() {
-    this.currentUser = this.userService.getUserById2(sessionStorage.getItem("userid")).subscribe((response)=>{
+    this.userService.getUserById2(sessionStorage.getItem("userid")).subscribe((response)=>{
```

Comment:
Why are there two getUserById methods?

Reply:
do you mean, why am I using version 2? That is the service that the previous team was using. They have more than one in the user service. I can try and remove it, but not sure I should be doing that.

Reply:
no need to do that in this PR but they should probably be consolidated